string cloud_stitch_req
---
string cloud_stitch_resp
